qemu/tests/cfg/virtio_console.cfg

```diff
@@ -228,10 +230,13 @@
                             # arm doesn't use pci bus
                             no aarch64
                             extra_params = " -device virtio-serial-pci,max_ports=32"
+                            extra_params_rhev = " -device virtio-serial-pci,max_ports=512"
```

Comment:
only qemu-kvm-rhev extend the max port to 512, right? qemu-kvm max port is still 32?